src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -149,6 +149,12 @@
   <data name="SettingsValidateErrorTitle" xml:space="preserve">
     <value>Encountered errors while loading user settings</value>
   </data>
+  <data name="KeyboardServiceWarningTitle" xml:space="preserve">
+    <value>Warning:</value>
+  </data>
+  <data name="KeyboardServiceWarningText" xml:space="preserve">
+    <value>The "Touch Keyboard and Handwriting Panel Service" isn't running on your machine. This can prevent the Terminal from receiving keyboard input.</value>

```

Comment:
Could use [GetServiceDisplayName](https://docs.microsoft.com/windows/win32/api/winsvc/nf-winsvc-getservicedisplaynamew) to query the localized name. Although the DisplayName in the Registry can be a MUI string reference, I tested that GetServiceDisplayName looks up the localized string so the caller need not do that.

Reply:
Ooh that's probably a good call, to have us substitute in the real localized name of it here.